docs/concepts/packaging-types.adoc

```diff
@@ -71,3 +71,23 @@ The hollow JARs are available under the following coordinates:
 </dependency>
 ----
 
+[NOTE]
+====
+anchor:hollow-jar-limitations[]Using pre-built hollow JARs has certain limitations:
```

Comment:
These are general limitations of all hollow JARs, not just the pre-built hollow JARs.